examples/electricity_demand_example.py

```diff
@@ -27,7 +27,7 @@
 import demandlib.bdew as bdew
 import demandlib.particular_profiles as profiles
 
-# The following dictionary is create by "workalendar"
+# The following dictionary has been created by "workalendar"
```

Comment:
```suggestion
# The following dictionary is created by "workalendar"
```
I am a bit puzzled here. First, you write that the dictionary has been/ is created using `workalendar` but later you define the holidays yourself.

Reply:
While I am writing comments here... :-)
Since those lines are commented out, I think this is just to suggest how to use an alternative library, without creating a new dependency for the installation of demandlib. My only suggestion would be to also name https://pypi.org/project/holidays/ as another option. Personally, it took me much too long to stumble across it.